### PR TITLE
tmate 2.2.0 (new formula)

### PR DESCRIPTION
**Pull request by nviennot**

### CODE REVIEW

**Comment by aristidesfl** [comment]
Closes https://github.com/ben-ng/minifyify/issues/115

File: Library/Formula/tmate.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+class Tmate < Formula
+  desc "Instant terminal sharing"
+  homepage "http://tmate.io"
+  url "https://github.com/tmate-io/tmate/archive/2.2.0.tar.gz"
+  sha256 "932148b24d9c67e524ce744c9480b5603c9f976afb6ad7ca6d63246cd9e7fd3a"
+
+  head "https://github.com/tmate-io/tmate.git"
+
+  bottle do
```
Comment (DomT4) [inline Library/Formula/tmate.rb]: Please don't add a bottle block. We'll add those automatically when/if merged.

File: Library/Formula/tmate.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    cellar :any
+    root_url "https://github.com/tmate-io/tmate.io/raw/gh-pages/cellar"
+    sha256 "15cd75a8cab65bd5ede8f329d94b1a561c2f44bb73fbcc25e7dd7474ad42fa90" => :el_capitan
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf"   => :build
```
Comment (DomT4) [inline Library/Formula/tmate.rb]: Cut the equal spacing here. It looks a bit neater but a lot of editors flag it and it's not worth that hassle.

File: Library/Formula/tmate.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  depends_on "automake"   => :build
+  depends_on "libtool"    => :build
+
```
Comment (DomT4) [inline Library/Formula/tmate.rb]: No need for the empty line.

File: Library/Formula/tmate.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    ENV.append "LDFLAGS", "-lresolv"
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/tmate", "-V"
```
Comment (DomT4) [inline Library/Formula/tmate.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (nviennot) [inline Library/Formula/tmate.rb]: tmate is a fork of tmux, and --long-options are not implemented. (same as https://github.com/Homebrew/homebrew/blob/master/Library/Formula/tmux.rb#L68)

Comment (DomT4) [inline Library/Formula/tmate.rb]: Alright, cool. Thanks for the quick reply.

**Comment by nviennot** [comment]
Hi Dominyk, thank you for reviewing the PR.

I've addressed all your comments :)

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by nviennot** [comment]
Thanks!

**Pull request closed without merging**